docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -111,6 +111,18 @@ $ sudo sed -r 's/group=".+"/group="kvm"/1' /etc/libvirt/qemu.conf > /etc/libvirt
 $ newgrp libvirt
 ----
 
+. Start the *default* libvirt network:
++
+----
+$ sudo virsh net-start default
+----
+
+. For convenience it's useful to mark the *default* network as autostart:
```

Comment:
convenience? I'd say this is rather essential.. but I thought the standard definition did that...

Reply:
@robin-owen This is a very generic topic... the only difference might be the location of the config file... ? right?

Reply:
I agree, this is generic information.

In the future, it would be nice if we can make use of AsciiDoc attributes to place these instructions for each distribution with the location of the configuration file, but that lies outside of the scope of this particular issue.